<launch>
	<!-- global rosparam -->
	<param name="use_sim_time" value="true"/>

	<!-- rosbag -->
	<node pkg="rosbag" type="play" name="player" args="--clock /home/amsl/ozaki/bagfiles/infant/indoor_dkan/2019-04-04-14-27-47.bag"/>

	<!-- rviz -->
	<node pkg="rviz" type="rviz" name="rviz" args="-d $(find velodyne_pointcloud_to_depthimage)/rviz_config/velodyne_pointcloud_to_depthimage.rviz"/>

	<!-- sensor driver -->
	<include file="$(find velodyne_pointcloud)/launch/32e_points.launch"/>

	<!-- main -->
	<node pkg="velodyne_pointcloud_to_depthimage" name="py_velodyne_pointcloud_to_depthimage" type="py_velodyne_pointcloud_to_depthimage.py" output="screen">
		<!-- <param name="num_ring" type="int" value="32"/> -->
		<!-- <param name="points_per_ring" type="int" value="1092"/> -->
		<!-- <param name="depth_resolution" type="double" value="0.1"/> -->
		<param name="save_limit" type="int" value="10"/>
		<param name="save_dir_path" type="string" value="$(find velodyne_pointcloud_to_depthimage)/saved"/>
		<!-- <param name="save_img_name" type="string" value="32e_"/> -->
	</node>
</launch>
